### ISSUE TITLE
Paper v3

### ISSUE THREAD

**Issue by bhack** [cybertronai/pytorch-lamb#2]
There was new clarification and details on v3 version of the paper (end of May).
take a look https://arxiv.org/abs/1904.00962v3 ?

**Comment by 8enmann** [comment]
I hadn't seen that, thanks! 
> Test Accuracy by MNIST training with LeNet (30 epochs for Batch Size = 1024). The
tuning space of learning rate for all the optimizers is {0.0001, 0.001, 0.01, 0.1}. We use the same
learning rate warmup and decay schedule for all of them.

Looks like I used almost the same hyperparameters. The only difference is they used learning rate warmup and decay, whereas in the example in this repo I'm using constant learning rate.

Feel free to reopen this if there's something I'm missing.
